timescaledb/how-to-guides/multinode-timescaledb/multinode-maintenance.md

```diff
@@ -23,6 +26,36 @@ $$;
 SELECT add_job('data_node_maintenance', '5m');
 ```
 
+## Maintaining statistics for distributed hypertables
+
+Table statistics need to be up-to-date to allow for efficient planning
+of queries on distributed hypertables. Unfortunately, due to their
+distributed nature, auto-vacuum is not useful to gather the statistics.
+So you need to explicitly ANALYZE your distributed hypertables
+periodically via a maintenance job as shown below.
+
+```sql
+CREATE OR REPLACE PROCEDURE distributed_hypertables_analyze(job_id int, config jsonb)
+LANGUAGE plpgsql AS
+$$
+DECLARE r record;
+BEGIN
+FOR r IN SELECT hypertable_schema, hypertable_name
+              FROM timescaledb_information.hypertables
+              WHERE is_distributed ORDER BY 1, 2
+LOOP
+EXECUTE format('ANALYZE %I.%I', r.hypertable_schema, r.hypertable_name);
+END LOOP;
+END
+$$;
+
+SELECT add_job('distributed_hypertables_analyze', '12h');
+```
+Note that both the above jobs can be merged into a single maintenance
+job if needed. However analyzing of distributed hypertables should
+be done at a much lesser frequency as compared to remote transaction
+healing activity.
```

Comment:
```suggestion

You can merge the jobs in this example into a single maintenance
job if you prefer. However, analyzing distributed hypertables should
be done less frequently than remote transaction healing activity.
```

This feels like it's missing some further explanation.

Reply:
Reworded and added one more line of explanation.